samples/cpp/benchmark/throughput_benchmark/README.md

```diff
@@ -0,0 +1,98 @@
+
+The following C++ API is used in the application:
+
+| Feature | API | Description |
+| :--- | :--- | :--- |
+| OpenVINO Runtime Version | `ov::get_openvino_version` | Get Openvino API version |
+| Basic Infer Flow | `ov::Core`, `ov::Core::compile_model`, `ov::CompiledModel::create_infer_request`, `ov::InferRequest::get_tensor` | Common API to do inference: compile a model, create an infer request, configure input tensors |
+| Asynchronous Infer | `ov::InferRequest::start_async`, `ov::InferRequest::set_callback` | Do asynchronous inference with callback. |
+| Model Operations | `ov::CompiledModel::inputs` | Get inputs of a model |
+| Tensor Operations | `ov::Tensor::get_shape`, `ov::Tensor::data` | Get a tensor shape and its data. |
+
+| Options | Values |
+| :--- | :--- |
+| Validated Models | [alexnet](@ref omz_models_model_alexnet), [googlenet-v1](@ref omz_models_model_googlenet_v1) [yolo-v3-tf](@ref omz_models_model_yolo_v3_tf), [face-detection-0200](@ref omz_models_model_face_detection_0200) |
+| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Supported devices | [All](../../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
+| Other language realization | [Python](../../../python/benchmark/throughput_benchmark/README.md) |
+
+## How It Works
+
+The sample compiles a model for a given device, randomly generates input data, performs asynchronous inference multiple times for a given number of seconds. Then processes and reports performance results.
+
+You can see the explicit description of
+each sample step at [Integration Steps](../../../../docs/OV_Runtime_UG/integrate_with_your_application.md) section of "Integrate OpenVINO™ Runtime with Your Application" guide.
+
+## Building
+
+To build the sample, please use instructions available at [Build the Sample Applications](../../../../docs/OV_Runtime_UG/Samples_Overview.md) section in OpenVINO™ Toolkit Samples guide.
+
+## Running
+
+```
+throughput_benchmark <path_to_model>
+```
+
+To run the sample, you need to specify a model:
+- You can use [public](@ref omz_models_group_public) or [Intel's](@ref omz_models_group_intel) pre-trained models from the Open Model Zoo. The models can be downloaded using the [Model Downloader](@ref omz_tools_downloader).
+
+> **NOTES**:
+>
+> - Before running the sample with a trained model, make sure the model is converted to the intermediate representation (IR) format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
+>
+> - The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
+
+### Example
+
+1. Install the `openvino-dev` Python package to use Open Model Zoo Tools:
+
+```
+python -m pip install openvino-dev[caffe,onnx,tensorflow2,pytorch,mxnet]
+```
+
+2. Download a pre-trained model using:
+
+```
+omz_downloader --name googlenet-v1
```

Comment:
Is it still ok to use OMZ tools in the samples or it is better to take models from public sources directly? @eaidova, can you please comment here?

Reply:
I think we can modify it later when we have such notebooks for many cases as it applies to all samples, not only one.